da/types/sidecars.go

```diff
@@ -27,43 +27,40 @@ import (
 	"github.com/sourcegraph/conc/iter"
 )
 
-// BlobSidecars is a slice of blob side cars to be included in the block.
-type BlobSidecars struct {
-	// Sidecars is a slice of blob side cars to be included in the block.
-	Sidecars []*BlobSidecar
-}
```

Comment:
@fridrik01 I wonder if we should keep this as a struct. I expect this to change the way we serialize sidecars, so maybe we want to keep it as it was?